spark/src/main/kotlin/com/adevinta/spark/components/image/Image.kt

```diff
@@ -148,6 +149,8 @@ internal fun SparkImage(
  * @param filterQuality Sampling algorithm applied to the image when it is scaled and drawn
  * into the destination. The default is [FilterQuality.Low] which scales using a bilinear
  * sampling algorithm
+ * @param withPlaceholder Don't show the shimmering placeholder in cases where the images is in fullscreen which
+ * would make a big flash when the image is loaded too quick (~less than 500ms)
```

Comment:
This documentation should be re-worded with the opposite meaning.
`withPlaceholder` default value is true, and requesting it would `show the shimmering placeholder`